components/admin_console/admin_definition.jsx

```diff
@@ -4808,15 +4807,9 @@ const AdminDefinition = {
                         defaultMessage='deprecated'
                     />
                 ),
-                shouldDisplay: (license) => license.IsLicensed && license.OpenId === 'true',
             },
             isHidden: it.any(
-                it.any(
-                    it.not(it.licensed),
-                    it.licensedForSku('starter'),
-                ),
                 it.all(
-                    it.licensedForFeature('OpenId'),
```

Comment:
Not sure if we need to update this definition because it is for oauth and not openID

Reply:
Hi Ben, OAuth will also be free here on. ref: https://community.mattermost.com/private-core/pl/1s98tpkuep8ejfabwsagmn93gy